leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -272,6 +278,7 @@ var LinkInspector = Backbone.View.extend({
             this.link.set('linkType', linkType);
             this.link.set('postType', postType);
             this.model.label(0, {position: 0.5, attrs: {text: {text: linkType + " | " + postType}}});
+            $('#link-type-end').val(this.link.get('postType'));
```

Comment:
I know its easier/more concise to put this line in here but since this function is really designed to alter cell properties, not html properties, I think it would be better to put it after you call setValues in each of the if statements.